src/pages/menu/components/optionModal/index.tsx

```diff
@@ -0,0 +1,161 @@
+/** @jsxImportSource @emotion/react */
+import { css } from "@emotion/react";
+import { useEffect } from "react";
+import colors from "styles/color";
+import { SubmitHandler, useFieldArray, useForm } from "react-hook-form";
+import type { CreateMenuOptionGroupReq } from "api/modules/menu/types";
+import { Button } from "common/components";
+import { Plus } from "common/components/icons";
+import {
+  NEW_OPTION_GROUP_TEMPLATE,
+  NEW_OPTION_ITEM_TEMPLATE,
+} from "pages/menu/constants";
+import { OptionModalProps } from "./types";
+import {
+  OnOffButton,
+  OptionItemInput,
+  LabelInput,
+} from "pages/menu/components";
+
+const OptionModal = ({
+  initial,
```

Comment:
조금 더 명확하게 표현되도 좋을 것 같습니다!
OptionModal 컴포넌트를 사용하는 쪽에서 initial props를 보면 어떤 걸 초기화하는지 헷갈릴 수 있을 것 같네요!
```suggestion
  initialFormValues,
```